marimo/_cli/export/commands.py

```diff
@@ -335,7 +341,75 @@ def export_callback(file_path: MarimoPath) -> ExportResult:
     return watch_and_export(MarimoPath(name), output, watch, export_callback)
 
 
+@click.command(
+    help="""Export a notebook as a WASM-powered standalone HTML file.
+
+Example:
+
+  \b
+  * marimo export html-wasm notebook.py -o notebook.wasm.html
+
+The exported HTML file will run the notebook using WebAssembly, making it
+completely self-contained and executable in the browser.
+
+In order for this file to be able to run, it must be served over HTTP,
+and cannot be opened directly from the file system (e.g. file://).
+"""
+)
+@click.option(
+    "-o",
+    "--output",
+    type=str,
+    required=True,
+    help="""Output directory to save the HTML to.""",
+)
+@click.option(
+    "--mode",
+    type=click.Choice(["edit", "run"]),
+    help="Mode to run the notebook in.",
+    required=True,
+)
+@click.option(
+    "--include-assets/--no-include-assets",
+    default=True,
+    show_default=True,
+    type=bool,
+    help="""
+    Include assets in the exported notebook.
+    These assets are required for the notebook to run in the browser.
```

Comment:
If the assets are required, why would someone choose to not include them? Can we get rid of this option?

Reply:
they may want to run this across a bunch and not keep copying the assets. i can remove this for now